src/components/Map/index.js

```diff
@@ -49,7 +55,7 @@ const LeaftletMap = ({markerCoordinates, onMarkerSet, disabled = false}) => {
       <TileLayer url={TILE_LAYERS.url} {...TILE_LAYERS.options} />
       {coordinates ? (
         <>
-          <MapView coordinates={coordinates} />
+          <MapView coordinates={defaultCenter} zoomLevel={defaultZoomLevel} />
```

Comment:
Double check if this doesn't reset the center view and zoom level when user places a marker.